dkist/utils/inventory.py

```diff
@@ -60,19 +64,24 @@ def __missing__(self, key):
 })
 
 
+def _key_clean(key):
```

Comment:
I thought we wanted to avoid having to define this here?

Reply:
Yeah, we did, but my little idea of making a dict full of `None`s didn't work.